common/src/main/java/com/viaversion/viabackwards/protocol/v1_19to1_18_2/rewriter/EntityPacketRewriter1_19.java

```diff
@@ -196,7 +201,12 @@ public void register() {
                 map(Types.BOOLEAN); // Debug
                 map(Types.BOOLEAN); // Flat
                 map(Types.BOOLEAN); // Keep player data
-                read(Types.OPTIONAL_GLOBAL_POSITION); // Read last death location
+                handler(wrapper -> {
+                    final GlobalBlockPosition lastDeathPosition = wrapper.read(Types.OPTIONAL_GLOBAL_POSITION);
+                    if (lastDeathPosition != null) {
+                        wrapper.user().put(new LastDeathPosition(lastDeathPosition));
+                    }
```

Comment:
This needs to handle null values as a reset, also above

Reply:
Done